extensions/amp-story/1.0/_locales/README.md

```diff
@@ -0,0 +1,59 @@
+# Supported languages
+
+To see a list of the supported languages, see the list of `*.js` files in the [`_locales` directory](https://github.com/ampproject/amphtml/tree/master/extensions/amp-story/1.0/_locales).
+
+# Language fallbacks
+
+Languages will fall back to their more general variants, and ultimately to the `default` language.  Variants are indicated by a hyphen in the language code.
+
+For example, the language code `en-GB` will check the following languages (in this order):
+
+* `en-GB` (English, Great Britain)
+* `en` (English)
+* `default` (Default)
+
+The `default` language code is a fallback used in the case that the publisher-specified language code does not exist.  It uses a minimal amount of English strings, so that the document can be displayed mostly in its primary language.  Any labels that describe otherwise familiar or intelligible iconography can be dropped entirely.  For example, since sharing icons show the sharing network's logo (e.g. the Twitter logo), the "Twitter" string is redundant and can be left out of the `default` language.
+
+# Viewing the current strings
+
+You can view the translations provided for each language in [this spreadsheet](https://bit.ly/amp-story-strings).  Any cells with the text `undefined` mean that the string will not be shown in the specified language, and any fall
```

Comment:
Looks like the last sentence was cut off?

Reply:
Whoops!  Fixed.